capabilities/communication/send-email/maps/mailchimp.suma

```diff
@@ -6,7 +6,7 @@ provider = "mailchimp"
 
 map SendEmail {
   body = {
-    key: input.mailchimp_api_key,
+    key: parameters.API_KEY,
```

Comment:
Key parameter is added to the body through security scheme and `API_KEY` integration parameter is not defined in mailchimp provider. This line should be omitted.

Reply:
Thats leftover. When I learned security schemes works with bodies as well. Will remove it. Thanks.

Reply:
Fixed